pipenv/vendor/vendor.txt

```diff
@@ -4,7 +4,7 @@ colorama==0.4.6
 dparse==0.6.3
 markupsafe==2.1.3
 pexpect==4.8.0
-pipdeptree==2.13.0
+pipdeptree==2.8.0
```

Comment:
Just curious, why is that you reverted pipdeptree?

Reply:
We have a custom patch that needs to be re-worked, it couldn't be applied  and I didn't have time to dig deeper.